src/stories/packages/components/ButtonOutline.stories.ts

```diff
@@ -46,9 +46,16 @@ const meta: Meta<typeof ButtonOutline> = {
 export default meta;
 type Story = StoryObj<typeof ButtonOutline>;
 
-export const Primary: Story = {
+export const Outline: Story = {
   args: {
     tooltip: 'This can provide additional context',
     children: 'Cancel',
   },
+  parameters: {
+    design: {
```

Comment:
can we list more then one figma link per component or in other storybook documentation blocks?

Reply:
I haven't seen an example of that in their documentation, but it's something we can explore.

Reply:
Ah, it turns out that you can pass an array to design! I'll let Callie know about that.

![image](https://github.com/DataBiosphere/terra-ui/assets/484484/7c2bed79-2619-4b77-a7e8-3bc345d240ec)

![image](https://github.com/DataBiosphere/terra-ui/assets/484484/f918207c-fd30-497a-a968-6bb36dd3abf6)